RobotAppEyes/RobotAppEyes.py

```diff
@@ -27,11 +27,43 @@
 from applitools.eyes import Eyes, BatchInfo
 from applitools.utils import _image_utils
 from applitools._webdriver import EyesScreenshot
+from applitools.target import Target, IgnoreRegionBySelector, FloatingRegionBySelector, FloatingBounds
 from version import VERSION
 
 _version_ = VERSION
 
 
+def get_search_element(selector):
+    if selector.upper() == 'CSS SELECTOR':
```

Comment:
We can simplify this a little bit I think. We could use a mapping:
```
seach_element_mapping = {
    'CSS_SELECTOR': By.CSS_SELECTOR,
    'XPATH': By.XPATH,
    'ID': By.ID,
    # Etc...
}

def get_search_element(selector):
    selector = selector.upper()  # Do this only once
    if selector in seach_element_mapping:
        return search_element_mapping[selector]
    raise InvalidElementStateException('...')
```
Or if we want to always EXACTLY match the attribute we could simply try this:
```
def get_search_element(selector):
    try:
        return getattr(By, selector.upper())
    except AttributeError:
        raise InvalidElementStateException('...')
```